recipes/gcta/meta.yaml

```diff
@@ -1,26 +1,26 @@
 {% set name = "gcta" %}
-{% set version = "1.93.2beta" %}
+{% set version = "1.94.1" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
-source:
-  - url: https://cnsgenomics.com/software/{{ name }}/bin/{{ name }}_{{ version }}.zip # [linux]
-    sha256: 44354f4909666b760ab3cf538dc143ee3feff5540c2ad237a098766ff9829374 # [linux]
-  - url: https://cnsgenomics.com/software/{{ name }}/bin/{{ name }}_{{ version }}_mac.zip # [osx]
-    sha256: b92f4ac51e5e5cb04c8eb93d07605efb9f8847ec0afcb8df2717200f995b6fe7 # [osx]
+source: https://yanglab.westlake.edu.cn/software/gcta/bin/gcta-1.94.1-linux-kernel-3-x86_64.zip
+  - url: https://yanglab.westlake.edu.cn/software/{{ name }}/bin/{{ name }}-{{ version }}-linux-kernel-3-x86_64.zip # [linux]
+    sha256: f2fc3f46b98f5eb2bbe0886a2c60268c6b634583f7a3fed51cfeb15e9f33fb0d # [linux]
+  - url: https://yanglab.westlake.edu.cn/software/{{ name }}/bin/{{ name }}-{{ version }}-macOS-x86_64.zip # [osx]
+    sha256: 51c2a5c622cb7ab8d246252abece9d28bf6e41c0e54a0b820a0871695a269441 # [osx]
 
 build:
-  number: 1
+  number: 0
 
 test:
   commands:
     - gcta64 | grep "Analysis started"
 
 about:
-  home: "https://cnsgenomics.com/software/gcta/"
-  license: MIT
+  home: "https://yanglab.westlake.edu.cn/software/gcta"
+  license: GPLv3
```

Comment:
Can you please use SPDX identifiers, eg. https://spdx.org/licenses/GPL-3.0-only.html

Reply:
Yes, sure